app/javascript/src/javascripts/blacklists.js

```diff
@@ -192,8 +117,8 @@ Blacklist.postHide = function (post) {
 
 Blacklist.postShow = function (post) {
   const $post = $(post);
-  Blacklist.postRestoreSrc(post);
-  $post.addClass("blacklisted").removeClass("blacklisted-active").removeClass("blacklisted-active-visible");
+  $post.removeClass("blacklisted");
+  Post.resize_notes();
```

Comment:
When the blacklist is active on load notes are scaled against 0x0 so I resize them here after the post has it's size restored.